src/ray/raylet/scheduling_queue.cc

```diff
@@ -395,6 +408,22 @@ std::string SchedulingQueue::DebugString() const {
   return result.str();
 }
 
+void SchedulingQueue::RecordMetrics() const {
+  for (const auto &task_state : {
+           TaskState::PLACEABLE,
+           TaskState::WAITING,
+           TaskState::READY,
+           TaskState::RUNNING,
+           TaskState::INFEASIBLE,
+           TaskState::WAITING_FOR_ACTOR_CREATION,
+       }) {
+    stats::SchedulingQueueStats().Record(
+        static_cast<double>(GetTaskQueue(task_state)->GetTasks().size()),
+        {{stats::ValueTypeKey,
+          std::string("num_") + GetTaskStateString(task_state) + "tasks"}});
```

Comment:
```suggestion
          std::string("num_") + GetTaskStateString(task_state) + "_tasks"}});
```

Reply:
does `GetTaskStateString` return upper case?

Reply:
It returns lowercases.